src/app/(home)/page.tsx

```
@@ -2,13 +2,13 @@ import Features from "@/components/Features";
 import Pricing from "@/components/Pricing";
 import Testimonials from "@/components/Testimonials";
-import { auth } from "@/lib";
+import { auth } from "@/lib/auth";
 
 export default async function Home() {
-  const { user } = await auth();
+  const { userId } = await auth();
   return (
     <>
-      <Hero authenticated={user ? true : false} />
+      <Hero authenticated={userId ? true : false} />
```

Biome (1.9.4)

[error] Unnecessary use of boolean literals in conditional expression.

Simplify your code by directly assigning the result without using a ternary operator.
If your goal is negation, you may use the logical NOT (!) or double NOT (!!) operator for clearer and concise code.
 Check for more details about NOT operator.
Unsafe fix: Remove the conditional expression with


(lint/complexity/noUselessTernary)